utils/build_rpms_from_scratch.sh

```diff
@@ -60,15 +134,62 @@ setup_rpm_build_env() {
     dnf config-manager --set-enabled powertools
   elif [[ "$el_version" == "9" ]]; then
     dnf config-manager --set-enabled crb # CodeReady Builder
-  else
-    echo "Unsupported EL Version: $el_version"
-    exit 1
   fi
 
   # ignore dpdk because want to use rpms build from daos-stack/dpdk
   dnf update -y --exclude=dpdk --exclude=dpdk-devel
 }
 
+l_createrepo() {
+  createrepo "$1"
+
+  if [[ "$OS_ID" == "opensuse-leap" ]]; then
+    zypper refresh
+  fi
```

Comment:
As mentioned earlier, you could use `dnf … makecache` here instead of using `--refresh` elsewhere.